src/components/CommentForm.js

```diff
@@ -86,9 +93,11 @@ export default class CommentForm extends React.Component {
               content={text}
               onBlur={() => updateStore(this.refs.editor.getContent())}
               onChange={setText}
-              onKeyDown={quickSubmit}/>
-            <input type='submit' value='Comment' ref='button'/>
+              onKeyDown={keyDown}/>
+            <input type='submit' value='Post' ref='button'
+              className={cx({enabled})}/>
             {close && <button onClick={close}>Cancel</button>}
+            <span className='meta help-text'>or press 'CMD + Enter' or 'CTRL + Enter' to post.</span>
```

Comment:
on Windows it's actually the Windows key, not Ctrl: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/metaKey

Maybe we should enable it for Ctrl as well.